torchci/lib/drciUtils.ts

```diff
@@ -24,14 +29,73 @@ Note: Links to docs will display an error until the docs builds have been comple
 }
 
 export function formDrciComment(
-    pr_num: number,
-    pr_results: string = "",
+  pr_num: number,
+  pr_results: string = "",
+  sevs: string = ""
 ): string {
-    const header = formDrciHeader(pr_num);
-    const comment =
-        `${DRCI_COMMENT_START}
+  const header = formDrciHeader(pr_num);
+  const comment = `${DRCI_COMMENT_START}
 ${header}
+${sevs}
 ${pr_results}
 ${DRCI_COMMENT_END}`;
-    return comment;
+  return comment;
+}
+
+
+export async function getDrciComment(
+  octokit: Octokit,
+  owner: string,
+  repo: string,
+  prNum: number
+): Promise<{ id: number; body: string }> {
+  const commentsRes = await octokit.rest.issues.listComments({
+    owner: owner,
+    repo: repo,
+    issue_number: prNum,
+  });
+  for (const comment of commentsRes.data) {
+    if (comment.body!.includes(DRCI_COMMENT_START)) {
+      return { id: comment.id, body: comment.body! };
+    }
+  }
+  return { id: 0, body: "" };
+}
+
+export function getActiveSEVs(issues: IssueData[]): string {
+  const activeSEVs = issues.filter(
+    (issue: IssueData) => issue.state === "open"
```

Comment:
For this, we will only care about open sev.  Do you think it's better to have this as a Rockset param with a default set to `open`?  The function name is `getActiveSEVs` anyway

Reply:
some other pages use the same sql query but want issues that arent closed.  I could make a new sql query, but it seems less confusing to just reuse the other one, but if rockset traffic becomes a big issue, this can be changed